### ISSUE TITLE
Tabs are too high

### ISSUE THREAD

**Issue by josvos** [gnunn1/tilix#1702]
Tabs are too high and take too much space (vertically).  This was also the case with Xfce terminals and IIRC the reason was some (changed) Gtk behavior.  In Xfce terminals, there is now a setting "Use custom styling to make tabs slim" (in Preferences / Appearance / Tabs) and when this option is turned on, tabs again have an elegant height (my reference: Xfce4 terminal version 0.8.7.4).  Can this option also be implemented in Tilix (or maybe even made the default)?

If needed, I can try to find out more details about how they implemented this feature.

I'm using Tilix 1.8.9 on Fedora 29.

**Comment by gnunn1** [comment]
If the tab height is the same height as you see for tabs in gedit then I have no plans to change it since I prefer sticking with the theme's look and feel. If the tilix tabs are bigger then the gedit tabs then that's an issue that should be fixed.

**Comment by josvos** [comment]
Never used gedit ;-) but I now installed and started it and yes, those tabs have the same height (as expected). See here for more discussion about this behavior and how it bothers more people using (GNOME/Xfce) terminal tabs: https://stackoverflow.com/questions/36869701/decrease-the-tabs-bar-height-in-gnome-terminal

Using the suggested `~/.config/gtk-3.0/gtk.css` "patch" does work btw, the tabs look then much better in Tilix, but it is a bit of a hassle.  As said, the Xfce terminal made it an option (default is also to keep standard height), to make it easy for users to make tabs look better.  It would be nice if Tilix would support something similar.  I found the code to do it in Xfce terminal source, let me know if you are interested (it's just adding the CSS code to a `gtk_css_provider`).
